tox.ini

```
@@ -56,6 +56,17 @@ commands =
     isort {posargs:{[tox]sources}}
     black {posargs:{[tox]sources}}
 
+[testenv:docs]
+changedir = docs
+extras =
+    dev
+    docs
```

I think the `dev` extra might not be necessary. Tested locally and this works for me:
```suggestion
extras = docs
```